app/services/analytics_events.rb

```diff
@@ -606,6 +614,65 @@ def idv_address_visit
     track_event('IdV: address visited') # rubocop:disable IdentityIdp/AnalyticsEventNameLinter
   end
 
+  # rubocop:disable Naming/VariableName,Naming/MethodParameterName
+  def idv_back_image_added(
+    width:,
+    height:,
+    mimeType:,
+    source:,
+    size:,
+    attempt:,
+    acuantCaptureMode:,
+    flow_path:,
+    acuant_sdk_upgrade_a_b_testing_enabled:,
+    use_alternate_sdk:,
+    acuant_version:,
+    fingerprint:,
+    phone_question_ab_test_bucket:,
+    failedImageResubmission:,
+    **_extra
```

Comment:
Is `**extra` unused on purpose?

Reply:
yeah, frontend events only accept keys that are defined so passing extra into the logging is not doing anything. I've updated the other frontend events accordingly.